src/core/PathosInstance.h

```diff
@@ -30,7 +30,11 @@ class PathosInstance : public Observable<ViewRequest>, Observer<Event> {
   std::unique_ptr<NcursesController> controller;
   std::unique_ptr<Map> map;
   std::unique_ptr<Player> player;
+
+  // Keep track of actionable position to indicate where the player is facing
   std::unique_ptr<Position> playerPos;
+  std::unique_ptr<Position> actionablePos;
```

Comment:
Consider this in all the movement events so we can use it for `CheckVicinityEvent`.